docs/sources/datasources/azuremonitor.md

```diff
@@ -413,3 +414,53 @@ Application Insights and Insights Analytics are two ways to query the same Azure
 > **Note** In Grafana 8.0, Application Insights and Insights Analytics will be deprecated and made read-only in favor of querying this data through Metrics and Logs. Existing queries will continue to work, but you cannot edit them.
 
 To prepare for this upcoming change, Application Insights queries can now be made in Metrics, under the "microsoft.insights/components" Namespace. Insights Analytics queries cannot be made within Logs with KQL at this time.
+
+## Query Azure Resource Graph service
```

Comment:
Add a short paragraph after this heading to provide context about what this service is and when or why you would want to query it.